stravalib/client.py

```diff
@@ -734,7 +734,7 @@ def upload_activity(self, activity_file, data_type, name=None, description=None,
         if not hasattr(activity_file, 'read'):
             if isinstance(activity_file, str):
                 activity_file = BytesIO(activity_file.encode('utf-8'))
-            elif isinstance(activity_file, str):
+            elif isinstance(activity_file, bytes):
```

Comment:
Can you please also update the docstring of this function, so the parameter has the correct types `TextIOWrapper`, `str`, and `bytes`?

Reply:
@jsamoocha @gitexel What is an example of a file type format that would be read as bytes? (just for my understanding) are gpx and tcx read in that format? many thanks

Reply:
@jsamoocha done

Reply:
@lwasser when you generate or download a file with `.fit` format it will be `bytes` type, so it will be much easier to pass the bytes directly to `upload_acitvity`

Reply:
@lwasser when you generate or download a file with `.fit` format it will be `bytes` type, so it will be much easier to pass the bytes directly to `upload_acitvity`